bundle/jsky.app.ot/src/main/resources/jsky/app/ot/gemini/altair/AltairForm.jfd

```diff
@@ -169,7 +169,7 @@
       </void> 
       <void method="add"> 
        <object class="com.jformdesigner.model.FormComponent"> 
-        <string>jsky.util.gui.OptionWidget</string> 
+        <string>javax.swing.JRadioButton</string>
```

Comment:
I suggest to remove them all and check that things still build - they should. If you find some showstopper there, we regroup. 

We aren't planning on using JFD anymore, so I think getting rid of all references to it would be a good thing to do.